### ISSUE TITLE
Replace Taylor's Version singles with album versions

### ISSUE THREAD

**Issue by SubjunctiveQuaver** [Roshy10/taylors-version#74]
Taylor released a few singles before albums dropped, e.g. Love Story, You All Over Me and Mr Perfectly Fine. Currently, the single versions aren't replaced with the proper album versions; this could be an option for those who prefer everything to be from the proper album.

This will again be a problem with Wildest Dreams/This Love when 1989 TV releases, and I would like to have the option to automatically change each of these to the album version when the time comes!

**Comment by Roshy10** [comment]
Do you mean to replace originals with the TV album version instead of the TV single, or to replace existing TV singles with TV album versions? (or both?)

I could definitely do the first one quite easily by just changing the mapping for the singles once there's an album out (I believe wildest dreams and this love are the only 2 single it's using at the moment). 

The second one I don't think most people would appreciate by default, so I'd need to make a switch to let it be customised.

**Comment by SubjunctiveQuaver** [comment]
Yes, I suppose both options. Makes sense for the 2nd to not be by default, but I don't remember all the playlists that have been updated to the TV singles (so I would have to look manually) so an option would definitely be hugely helpful!
